.github/workflows/release-build.yml

```diff
@@ -22,7 +82,7 @@ jobs:
         with:
           job-id: jdk11
           remote-build-cache-proxy-enabled: false
-          arguments: build final closeAndReleaseSonatypeStagingRepository --stacktrace -x :smoke-tests:test -Prelease.version=${{ github.event.inputs.version }}
+          arguments: build final closeAndReleaseSonatypeStagingRepository --stacktrace -x test -Prelease.version=${{ github.event.inputs.version }}
```

Comment:
Let's either rename `build` -> `assemble` unless we're comfortable relying on build cache after adding `11` above

Reply:
I like assemble 👍